backend/server/services/UserInfoService.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file Defines the UserInfoService
+ */
+
+const admin = require('firebase-admin');
+const winston = require('winston');
+
+/**
+ * Creates a new canvas based on a UserInfoService request
```

Comment:
Are you actually making a new canvas here? seems like you are doing more with the user information.